medusa/__init__.py

```diff
@@ -748,8 +752,8 @@ def initialize(consoleLogging=True):  # pylint: disable=too-many-locals, too-man
         GIT_REMOTE_URL = check_setting_str(CFG, 'General', 'git_remote_url',
                                            'https://github.com/%s/%s.git' % (GIT_ORG, GIT_REPO))
 
-        if 'com/sickrage' in GIT_REMOTE_URL.lower():
-            GIT_REMOTE_URL = 'https://github.com/PyMedusa/SickRage.git'
+        if 'com/sickrage' in GIT_REMOTE_URL.lower() or 'sickrage.git' in GIT_REMOTE_URL.lower():
+            GIT_REMOTE_URL = 'https://github.com/pymedusa/Medusa.git'
```

Comment:
this is where we change current git_remote_url
the first part of the if was done in SICKRAGETV migration